packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -137,39 +161,54 @@ class Send extends Component {
     );
   }
 
+  preValidate = (values, withError) => {
```

Comment:
To remove 5 lines of code, we could remove the withError argument here.

In the decorator:
```javascript
       if (this.preValidate(allValues, false) === true) {
          ...
        } else {
          // this means amount has errors
          ...
        }
```

in validateAmount:
```javascript
if (preValidation !== true) {
  return preValidation;
}
```

Reply:
Thanks, that `withError` felt pretty bad indeed :)